salt/modules/ceph.py

```diff
@@ -118,7 +115,7 @@ def zap(target=None, **kwargs):
         The cluster UUID. Defaults to value found in ceph config file.
     """
     if target is not None:
-        log.warning("Depricated use of function, use kwargs")
+        log.warning("Deprecated use of function, use kwargs")
```

Comment:
My vote for reverting changes to this file *except* this one, because they are not strictly spelling related.

Reply:
I added commit 0bca4366663d98bae2dfbf02b4fa400e241383c2 (Drop Python 2 support from salt/modules/ceph.py) to address the failing test case. I can drop this commit again if you prefer having failed test cases.

Reply:
If a file is touched, it will get it's Py2 and six support removed automatically.
The failure to do so will make the PR fail.
So, while I agree that this no longer just a spelling fix, it's a necessary change.